plugin/etcd/msg/path_test.go

```diff
@@ -10,3 +10,15 @@ func TestPath(t *testing.T) {
 		}
 	}
 }
+
+func TestPath(t *testing.T) {
+	result1 := Domain("/skydns/local/cluster/staging/service/")
+	if result1 != "service.staging.cluster.local." {
+		t.Errorf("==>%s", result1)
+	}
+
+	result2 := Domain("/skydns/local/cluster/staging/service")
+	if result2 != "service.staging.cluster.local." {
+		t.Errorf("==>%s", result2)
+	}
```

Comment:
These errors should be more descriptive.

Reply:
This was marked resolved - but no changes were made.  The errors are non-descriptive and identical, so it's impossible to know which one failed if one does, and the other does not.   An error should _at the very least_ provide a means of uniquely identifying which test failed.  Better, would be to also show the expected result and the incorrect result. Best would be to also briefly describe what is being tested.  Please see other tests in the project as examples.